library/built-in/SDXL/_cushySDXLLayout.tsx

```diff
@@ -37,40 +41,74 @@ export function _cushySDXLLayout(): Maybe<DisplaySlotFn<$CushySDXLUI['$Field']>>
-                  const conditioningIcon: IconName = index == 0 ? 'mdiArrowDown' : 'mdiFormatListGroupPlus'
-                  return (
-                     <UY.Misc.Frame tw='flex items-center' hover key={item.id}>
-                        <span tw={['line-clamp-1 w-full flex-grow px-1', !item.active && 'opacity-50']}>
-                           {item.child.text}
-                        </span>
-                        <div tw='flex-none'>
-                           <IkonOf name={conditioningIcon} />
-                        </div>
-                        <div tw='w-2' />
-                        <div tw='flex-none'>
-                           {/* <InputNumberUI
+         Body: observer((p) => {
+            const positive = ui.field.value.positive
+            const activePrompt = positive.prompts[positive.activeIndex]
+            return (
+               <>
+                  <UY.list.BlenderLike<typeof p.field> //
+                     activeIndex={positive.activeIndex}
+                     field={p.field}
+                     renderItem={(item, index) => {
+                        const conditioningIcon: IconName =
+                           index == 0 ? 'mdiArrowDown' : 'mdiFormatListGroupPlus'
+                        return (
+                           <UY.Misc.Frame
+                              tw='flex items-center'
+                              hover
+                              key={item.id}
+                              onMouseDown={() => (positive.activeIndex = index)}
+                           >
+                              <span tw={['line-clamp-1 w-full flex-grow px-1', !item.active && 'opacity-50']}>
+                                 {item.child.text}
+                              </span>
+                              <div tw='flex-none'>
+                                 <IkonOf name={conditioningIcon} />
+                              </div>
+                              <div tw='w-2' />
+                              <div tw='flex-none'>
+                                 {/* <InputNumberUI
                               // TODO(bird_d/ui/logic): Implement showing strength based on the conditioning type, should only appear on blend/add/etc. concate doesn't need it for example.
                               mode='float'
                               hideSlider
                               onValueChange={() => {}}
                               value={ree}
                            /> */}
-                           <UY.Misc.Checkbox
-                              square // TODO(bird_d/ui): Buttons like this, where there's only an icon, should just automatically apply square if there's no text/children.
-                              toggleGroup='prompt'
-                              value={item.active}
-                              onValueChange={(v) => item.setActive(v)}
-                           />
-                        </div>
+                                 <UY.Misc.Checkbox
```

Comment:
do you want to hoist those `.Misc.` ?
`Button` and `Checkbox` prob deserve to be top-level in UY